arangod/IResearch/IResearchViewNode.cpp

```diff
@@ -286,7 +286,7 @@ aql::ExecutionBlock* IResearchViewNode::createExecutionBlock(
     LOG_TOPIC(WARN, IResearchFeature::IRESEARCH)
       << "failed to get view implementation while creating IResearchView ExecutionBlock";
 
-    return nullptr;
+    THROW_ARANGO_EXCEPTION(TRI_ERROR_INTERNAL);
```

Comment:
Throwing the TRI_ERROR_INTERNAL exception without any message is not really desired.
The problem is that when an internal error happens, there is no indication of what exactly went wrong. I suggest using the `THROW_ARANGO_EXCEPTION_MESSAGE` macro instead, which allows to pass a second parameter with a more descriptive error message.